frontend/apps/monkvision/main.html

```diff
@@ -117,9 +117,7 @@
 <body>
 <div id="leftheader">
 	{{#dashboards}}
-		{{#showDash}}
-			<img id="{{id}}" class="dashicon" src="{{{htmlData.dashicon}}}" alt="{{title}}" title="{{title}}" onclick="router.loadPage('./main.html?dash=./dashboards/{{file}}&title={{title}}&refresh={{refresh}}&name={{name}}');"><span>{{name}}</span>
-		{{/showDash}}
+		<img id="{{id}}" class="dashicon" src="{{{htmlData.dashicon}}}" alt="{{title}}" title="{{title}}" onclick="router.loadPage('./main.html?dash=./dashboards/{{file}}&title={{title}}&refresh={{refresh}}&name={{name}}');"><span>{{name}}</span>
```

Comment:
No need for a tab here - indentation is not required when it is clear you are traversing an array.